src/components/Menus/Sidebar/InstructionsPanel/ProgressBar/ProgressBar.test.js

```diff
@@ -66,10 +67,36 @@ describe("When on a middle step", () => {
 
 describe("When on last step", () => {
   beforeEach(() => {
-    renderProgressBarOnStep(2);
+    renderProgressBarOnStep(3);
   });
 
   test("Next step button is disabled", () => {
     expect(screen.getByTitle("instructionsPanel.nextStep")).toBeDisabled();
   });
 });
+
+describe("When on a quiz step", () => {
+  beforeEach(() => {
+    renderProgressBarOnStep(2);
+  });
+
+  test("Progress bar renders with correct max and value", () => {
+    const progressBar = screen.getByRole("progressbar");
+    expect(progressBar).toHaveValue(2);
+    expect(progressBar).toHaveAttribute("max", "3");
+  });
+
+  test("Clicking previous step button goes back a step", () => {
+    const previousStepButton = screen.getByTitle(
+      "instructionsPanel.previousStep",
+    );
+    fireEvent.click(previousStepButton);
+    expect(store.getActions()).toEqual([setCurrentStepPosition(1)]);
+  });
+
+  test("Clicking next step button goes forward a step", () => {
+    const nextStepButton = screen.getByTitle("instructionsPanel.nextStep");
+    fireEvent.click(nextStepButton);
+    expect(store.getActions()).toEqual([setCurrentStepPosition(3)]);
+  });
```

Comment:
Not sure I quite understand what's going on here... I might be missing something, but what indicates that the "Quiz step 1" step is a quiz?

Reply:
Yes you're right! I've fixed (I think I reverted some changes I made and removed the knowledgeQuiz property on that step). Thanks for spotting!

Reply:
I'm still not sure I understand... Does the progress bar know anything about whether it's a quiz?

Reply:
No it doesn't. 

 I put it in since the ProgressBar was previously removed on a quiz step but maybe this validation is unnecessary

Reply:
Hmmm yeah, tbh I was wondering whether the test was redundant 🤔 I guess in another file you've got the test that the project bar shows on a quiz step, so I think that's enough... I suppose all you've done is remove the logic to hide the progress bar on a quiz step, so if the tests were to go back to how they were before that was added, that would make sense in my mind at least 😅

Reply:
Thanks - I've removed!